Scripts/RunUnitTests.bat

```diff
@@ -6,6 +6,7 @@ IF "%1"=="" (SET "Configuration=Debug") ELSE (SET "Configuration=%1")
 set RESULT=0
 
 %VFS_OUTPUTDIR%\GVFS.UnitTests.Windows\bin\x64\%Configuration%\GVFS.UnitTests.Windows.exe  || set RESULT=1
+dotnet %VFS_OUTPUTDIR%\GVFS.UnitTests.Mac\bin\x64\%Configuration%\netcoreapp2.1\GVFS.UnitTests.Mac.dll  || set RESULT=1

```

Comment:
Should this be in a Mac-specific script? This will only get run on Windows in CI/PRs.

Reply:
The Mac specific UT is removed.